packages/utils/upgrade/README.md

```diff
@@ -1 +1,179 @@
-<h1 align="center">upgrade</h1>
+# Strapi Upgrade Tool
+
+## Description
+
+The Strapi Upgrade Tool is the CLI for facilitating upgrades between Strapi versions.
+
+When run, the command will "step" from your current version of Strapi to the next version that includes code tranforms for breaking changes.
```

Comment:
This line will likely change before release but for now it's the plan.

Reply:
Confirmed (the flow will be different), but we can keep it for now